tests/e2e/render-utils.sh

```diff
@@ -866,6 +866,17 @@ if [ ! -z "$output" ]; then
     IS_OPENSHIFT=true
 fi
 
+export KAFKA_USE_CUSTOM_PODSET
+if [ -z "$KAFKA_VERSION" ]; then
```

Comment:
Can we get rid of the nested if with something like this?
```suggestion
if [ -z "$KAFKA_VERSION" ] || [ version_gt $KAFKA_VERSION "0.25.0" ]; then
```